multitenancy/tenant-manager-api/src/main/java/io/apicurio/multitenant/api/RegistryTenantResource.java

```diff
@@ -66,8 +66,6 @@ public Response createTenant(NewRegistryTenantRequest tenantRequest) {
             tenant.setAuthServerUrl(defaultAuthServerUrl.get());
```

Comment:
I think this can be removed and fall back to the default auth URL configured in Registry without the need of also configure it in the tenant management API.